Sources/PackageDescription/Version.swift

```diff
@@ -56,7 +56,11 @@ extension Version: Hashable {
         if let build = buildMetadataIdentifier {
             result = (result &* mul) ^ UInt64(build.hashValue)
         }
-        return Int(bitPattern: UInt(result))
+        #if arch(i386) || arch(arm)
```

Comment:
I would argue that a conditional may not be required.  Using truncatingBitPattern should be equivalent to bitPattern if truncation is not required.